packages/cxl-ui/src/components/cxl-certificate-header.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, nothing } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import '@conversionxl/cxl-ui/src/components/cxl-section.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+
+import cxlCertificateHeaderStyles from '../styles/cxl-certificate-header-css.js';
+import cxlCertificateHeaderGlobalStyles from '../styles/global/cxl-certificate-header-css.js';
+
+@customElement('cxl-certificate-header')
+export class CXLCertificateHeaderElement extends LitElement {
+  static get styles() {
+    return [cxlCertificateHeaderStyles];
+  }
+
+  @property({ type: String }) theme = '';
+
+  @property({ type: String }) title = '';
+
+  @property({ type: String }) description = '';
+
+  @property({ type: Boolean, attribute: 'hide-actions' }) hideActions = false;
+
+  @property({ type: Boolean, attribute: 'hide-stats' }) hideStats = false;
+
+  @property({ type: Boolean, attribute: 'hide-credential' }) hideCredential = false;
+
+  // eslint-disable-next-line class-methods-use-this
+  _renderCompleted() {
```

Comment:
only "completed" looks like an event handler

```suggestion
  _renderCompletedIcon() {
```

Reply:
@pawelkmpt Renamed it to _renderCompletedEmblem, because it is not only icon. Is it ok?